web/src/auth/LoginPage.js

```diff
@@ -283,11 +283,11 @@ class LoginPage extends React.Component {
                 {i18next.t("login:Auto login")}
               </Checkbox>
             </Form.Item>
-            <a style={{float: "right"}} onClick={() => {
+            <button style={{float: "right"}} className="link-button" onClick={() => {
```

Comment:
Your code doesn't have hyperlink highlight, see: https://door.casbin.com/login

![image](https://user-images.githubusercontent.com/3787410/126884941-54d0707d-8f97-4a60-bcd8-7c3636000907.png)

Reply:
Already fixed

Reply:
@sh1luo  Harding-coding the colors is not the correct way. Plz rollback these a tag changes.